controllers/orderController.js

```diff
@@ -23,17 +23,18 @@ const createOrder = async (req, res) => {
 //initial state
   const initialValue = { subtotal: 0, orderItems: [] };
 
+  // this reduce function is probably worth refactoring/extracting into its own function since its somewhat long.
```

Comment:
@akosasante Akos, ehat do you mean exactly when say // this could potentially also be extracted to a helper function
  const { name, price, image, _id } = dbProduct;
  const singleOrderItem = {
    amount: item.amount,
    name,
    price,
    image,
    product: _id,
  };  should I export in to another document?

Reply:
@akosasante got it!!! thank you so much, that's was hard to understand at first but now I see it much more clear